packages/shared/src/lib/core/ledger/utils/handleLedgerError.ts

```diff
@@ -12,6 +12,8 @@ import { LedgerAppName, LedgerError } from '../enums'
 import { deriveLedgerError } from '../helpers'
 import { checkOrConnectLedger, ledgerPreparedOutput, resetLedgerPreparedOutput } from '@core/ledger'
 import { sendOutput } from '@core/wallet'
+import { activeProfile } from '@core/profile/stores'
```

Comment:
Im not sure we can rely on the active profile here? Because we might be during onboarding when we get an error. But maybe this still works as it falls back to shimmer?